index.html

```diff
@@ -123,9 +123,10 @@ <h5 class="topic1">Exhibition</h5>
             </div>
         </div>
     </div>
-        <button>Join BC World Summit 2022</button>
+        <button class="button">Join BC World Conference 2022</button>
         <a class="p-link" href="#">See whole program</a>
     </section>
+    <section id="fsp"></section>
```

Comment:
This section is missing the divider below the section header as well.

<img width="1042" alt="Screen Shot 2022-07-27 at 9 17 14" src="https://user-images.githubusercontent.com/82426682/181270336-526cc5a9-2a61-4498-ba5a-1628d159580a.png">